src/components/common/CaseDetailsForms/SingleNote.js

```diff
@@ -0,0 +1,87 @@
+import { Button } from 'antd';
+import React, { useState } from 'react';
+
+function SingleNote(props) {
+  const note = props.props;
+
+  const containerStyle = {
+    borderStyle: 'groove',
+    background: '#3f3f3f',
+    color: '#fcfcfc',
+    margin: '0 auto 1rem',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(5, 1fr)',
+    gap: '5px',
+    gridAutoFlow: 'row dense',
+    width: '50vw',
+  };
+
+  const contentStyle = {
+    gridRow: '2',
+    gridColumn: '2 / span 4',
+    paddingRight: '1rem',
+  };
+
+  const buttonStyle = {
+    gridRow: '3',
+    gridColumn: '5',
+    margin: '1rem',
+    color: '#CDCDCD',
+    background: '#9E5291',
+    borderColor: '#9E5291',
+  };
+
+  const infoStyle = {
+    margin: '.5rem 0',
+    gridRow: '1',
+    gridColumn: '3 / span 2',
+  };
+
+  const pictureStyle = {
+    gridRow: '2',
+    gridColumn: '1',
+    // marginLeft: '3rem',
```

Comment:
should we delete the commented out code?

Reply:
good catches, I went ahead and removed the commented out code in each component you pointed out.